### ISSUE TITLE
Changes not saving on set up page

### ISSUE THREAD

**Issue by balucanb** [widapro/wledPixel#30]
Struggling to get this to work. running the newest version. When I enter the MQTT zone text topic and change text to something else it does not save and reverts to text. Also when I enter the mqtt user , the home assistant long lived token and the http or https they will not save either. I have changed all of the above and clicked apply and I get a success message but as soon as I go back into the page all the changes disappear or revert.

**Comment by widapro** [comment]
Please check out this version: [wledPixel-v2.5.2b_WeMos_D1_mini.esp8266.bin.zip](https://github.com/widapro/wledPixel/files/10810232/wledPixel-v2.5.2b_WeMos_D1_mini.esp8266.bin.zip)

**Comment by balucanb** [comment]
Thanks I will let you know how it works

**Comment by balucanb** [comment]
Ok that seems to have corrected the issue! everything is remaining saved and the matrix is receiving data now thanks! Question though the previous version under "Display settings" had 4 zones, this version only has 3? Was that part of the fix or something else? Sign is getting info and so far (last 5 minutes) it appears to be working correct.

**Comment by widapro** [comment]
I temporarily turned off the ability to use 4 zones. Depending on the operating modes of the zones, sometimes there were crashes due to lack of memory.

If this beta version works stably, then in the next releases I plan to return the 4th zone.

**Comment by balucanb** [comment]
Spoke to soon. Worked all day fine, 8 hours-ish then it just crashed, display was just showing random blocks lighting up. ended up needing to erase the flash then re flash the bin file you sent. display shows the ip but I can't get it (the ip) it will sometimes partially load the page, I got to the setting page once then started getting the fail message when I tried to update anything. Wifi signal is great, I see the display on my network, ect. not sure where to go from here.
